core/src/main/kotlin/io/customer/sdk/core/util/FallbackLogger.kt

```diff
@@ -0,0 +1,20 @@
+package io.customer.sdk.core.util
+
+import io.customer.sdk.core.di.SDKComponent
+
+/**
+ * A fallback in case the SDK failed to initialize [Logger] instance before any logging calls happen.
+ *
+ * This allows us to expose the logger as non-null type without assuming any default logging levels.
+ *
+ * If any logs of this class are seen, that means there is a bug with SDK initialization logic for the [Logger]
+ */
+internal class FallbackLogger : LoggerImpl(SDKComponent.DEFAULT_LOG_LEVEL) {
+    init {
+        error(
```

Comment:
this will log error of not initialized for wrappers, a new error that the CS might not be aware of.